src/transaction/log_page_buffer.c

```diff
@@ -1088,30 +739,27 @@ logpb_create (THREAD_ENTRY * thread_p, LOG_PAGEID pageid)
  *   pageid(in): Page identifier
  *   fetch_mode(in): Is this a new log page ?. That is, can we avoid the I/O
  *
- * NOTE:Fetch the log page identified by pageid into a log buffer and
- *              return such buffer. The page is guaranteed to stay cached in
- *              the buffer until the page is released by the caller.
- *              If the page is not resident in the log buffer pool, the oldest
- *              log page that is not fixed is replaced. The rationale for this
- *              is that if we need to go back to rollback other transactions,
- *              it is likely that we need the earliest log pages and not the
- *              oldest one. A very old page may be in the log buffer pool as a
- *              consequence of a previous rollback. This does not provide
- *              problems to the restart recovery since we hardly go back to
- *              the same page.
+ * NOTE: first, the function checks if the pageid is the header page id.
+ *       if it is not, it brings the coresponding page from the log page buffer.
+ *       if the actual pageid differs from the buffer pageid, it means that it should
+ *       be invalidated - it contains another page - and it is flushed to disk, if it
+ *       is dirty. Now, if the pageid is null , we have a clear log page. If the fetch
+ *       mode is NEW_PAGE, it set the fields in the buffer, else, if it is OLD_PAGE, 
+ *       it brings the page from the disk. The last case is if the page is not NULL, and
+ *       it is equal with the pageid. This means it is an OLD_PAGE request, and it returns
+ *       that page.
  */
 static LOG_PAGE *
 logpb_fix_page (THREAD_ENTRY * thread_p, LOG_PAGEID pageid, PAGE_FETCH_MODE fetch_mode)
```

Comment:
@funkydvd @ribram 

I'd like to propose to rename the function, because we have removed `logpb_unfix_page`. Generally, fix-unfix, lock-unlock works together. 
I have thought about `logpb_locate_page` and `logpb_load_page`. I prefer to the first one but I don't really love it. Can you think of a better name? :)